### PR TITLE
Enable rubocop for spec files

### PR DESCRIPTION
**Pull request by jkbe**
Rubocop currently doesn't perform coding style checks on our spec files. However, as we are practicing TDD, tests are an important part of our work and therefore should be checked for our coding style as well (as discussed in meeting of Scrum Masters).

### CODE REVIEW

File: spec/controllers/teams_controller_spec.rb [hunk 1]
```diff
@@ -308,17 +308,17 @@
...
+        team.owners << another_user
+        team.owners << still_another_user
 
-      expect {
-        post :perform_action_on_multiple_members, params: { id: team.id, members: [another_user.id, still_another_user.id], delete_ownership: "delete_ownership"}
-      }.to change(team.owners, :count).by(-2)
+        expect {
+          post :perform_action_on_multiple_members, params: { id: team.id, members: [another_user.id, still_another_user.id], delete_ownership: "delete_ownership" }
+        }.to change(team.owners, :count).by(-2)
       end
```
Comment (fpottbaecker) [inline spec/controllers/teams_controller_spec.rb]: this looks to be indented too much

**Review by fpottbaecker — COMMENTED** [review]
Did you discuss the matter of quotation consistency (suggested: use single quotes unless interpolation or escaping is required [this is rubocop default])?

File: spec/controllers/teams_controller_spec.rb (hunk 1 above)
Comment (jkbe) [reply spec/controllers/teams_controller_spec.rb]: Yup, just fixed this. Weird autocorrect.

**Comment by jkbe** [comment]
@Pottiman I believe we have not yet discussed this although I'd prefer a uniform style here as well. I'll put it on the agenda for Friday.

**Review by fpottbaecker — APPROVED** [review]
Ideally i would want to put off merging until Friday, once the quotation style is chosen

**Comment by jkbe** [comment]
Hm, I think we can also change the configuration step by step, doesn't make that big of a difference.

**Comment by jkbe** [comment]
@hpi-swt2/scrum-masters fyi

**Pull request merged**